src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/express/ExpressCaseServiceTest.java

```diff
@@ -92,6 +101,15 @@ private static Stream<Arguments> provideIsExpressCase() {
         );
     }
 
+
+    private static Stream<Arguments> provideIsExpressCaseFinRemCaseDate() {
```

Comment:
I was admiring your nice test data creation....and then I read the method name 😉